### PR TITLE
INTGR-160

### PR DESCRIPTION
**Pull request by kmets89**

### CODE REVIEW

File: src/components/CodeSamplesContainer/CodeSampleStyle.tsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export default function CodeSampleStyle(): Record<string, string> {
+    return {
+        fontSize: "12px",
+        fontWeight: "400",
+        lineHeight: "16px",
+        textAlign: "left"
```
Comment (jyalenkatian) [inline src/components/CodeSamplesContainer/CodeSampleStyle.tsx:6]: We're missing some of the formatting listed in Figma
```suggestion
        fontSize: "12px",
        fontWeight: "400",
        lineHeight: "16px",
        textAlign: "left",
        borderRadius: "8px",
        fontFamily: "Roboto",
        overflowY: "auto",
        padding: "24px"
```

File: src/variables.scss [hunk 2]
```diff
@@ -1,6 +1,7 @@
 $brightBlue: #0069BF;
 $darkBlue: #4E677E;
-$lightGray: #FFFFFF;
+$lightGray:#F9FAFB;
```
Comment (jyalenkatian) [inline src/variables.scss:3]: The gray background color for the header is slightly different than the one for the top of the page. Header is #F9FAFB while the top of the page that includes the breadcrumb and title should be #F8F9FC.

File: src/pages/pages.scss [hunk 3]
```diff
@@ -82,7 +86,8 @@ ul {
 .title {
   font-weight: 300;
   font-size: 28px;
-  line-height: 36px
+  line-height: 36px;
+  margin-block-start: 8px;
 }
```
Comment (jyalenkatian) [inline src/pages/pages.scss:91]:
```suggestion
  margin-block-end: 0px;
}
```

File: src/pages/pages.scss [hunk 4]
```diff
@@ -91,6 +96,25 @@ ul {
...
+  margin-block-end: 0px;
+}
+
+.breadCrumb span {
+  align-self: center;
+}
+
 a {
   color: $brightBlue;
 }
```
Comment (jyalenkatian) [inline src/pages/pages.scss:120]: Would be better to call the .grayBackground class something else (i.e. pageHeading) and it should also have a padding of "16px 24px" while the title margin-block-end should be 0px.

**Review by jyalenkatian — CHANGES_REQUESTED** [review]

**Review by williamwgeotab — COMMENTED** [review]
Minor suggestions

File: src/components/CodeSamplesContainer/CodeSampleStyle.tsx [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
+    };
```
[the remark this answers is not in the document]
Comment (williamwgeotab) [reply src/components/CodeSamplesContainer/CodeSampleStyle.tsx]: Code sample styling will be revisited in the future

**Comment by williamwgeotab** [comment]
Latest commit looks good! 
- All functionality appear to be working fine
- CSS issues (naming, color code consistency, etc) will be addressed in the near future
- Newly merged "Using Custom Telematics Devices" doesn't have breadcrumbs but can be addressed in the future

Merging PR

**Pull request merged**